src/ray/raylet/node_manager.h

```diff
@@ -139,7 +181,12 @@ class NodeManager {
   /// this could grow unbounded.
   std::unordered_set<ClientID> removed_clients_;
   std::unordered_map<ClientID, TcpServerConnection> remote_server_connections_;
+  /// A mapping from actor ID to registration information about that actor
+  /// (including which node manager owns it).
   std::unordered_map<ActorID, ActorRegistration> actor_registry_;
+  /// A list of the actors on this node that have died. In principle, this could
+  /// grow unbounded.
+  std::unordered_set<ActorID> dead_actors_;
```

Comment:
It may make sense to fold this information into the `actor_registry_` to keep actor information centralized.

Reply:
Good idea